datasrc/network.py

```diff
@@ -268,6 +274,30 @@
 		NetIntRange("m_Precision", 0, 3),
 		NetFlag("m_RaceFlags", RaceFlags),
 	]),
+
+	NetObject("RaceFlag", [
+		NetIntAny("m_X"),
+		NetIntAny("m_Y"),
+		NetIntRange("m_Place", 'RACE_FLAG_MISSING', 'RACE_FLAG_BRONZE'),
+	]),
```

Comment:
I think this netobj isn't necessary when the race flags are always shown at tees' positions.

Reply:
@Robyt3 true, my thought process was adding this because modders might want to use this flags, too! Also i wanted to implement this as similar to the normal flags as possible